python/pipeline/fuse.py

```diff
@@ -0,0 +1,64 @@
+from distutils.version import StrictVersion
+import sys
+import datajoint as dj
+from . import experiment, reso, meso, shared
+from .exceptions import PipelineException
+
+assert StrictVersion(dj.__version__) >= StrictVersion('0.8.2'), "Please upgrade datajoint to 0.8.2+"
+
+
+schema = dj.schema('pipeline_fuse', locals())
+
+
+@schema
+class ScanDone(dj.Computed):
+    definition = """
+    # Calcium Activity from multiple pipelines
+    -> experiment.Scan
+    -> shared.PipelineVersion
+    -> shared.SpikeMethod
+    -> shared.SegmentationMethod
+    ---
+    pipe : char(8)   # meso, reso, etc
+    """
+
+    key_source = meso.ScanDone().proj() + reso.ScanDone().proj()
+
+    class Reso(dj.Part):
+        definition = """
+        -> reso.ScanDone
+        -> ScanDone
+        """
+
+    class Meso(dj.Part):
+        definition = """
+        -> meso.ScanDone
+        -> ScanDone
+        """
+
+    def _make_tuples(self, key):
+
+        for pipe, src, dest in (
```

Comment:
Could we do this check with ifs, the for else clause is a bit confusing:
```python
if reso.ScanDone() & key:
   pipe, src, dest = 'reso', reso.ScanDone, ScanDone.Reso
elif meso.ScanDone() & key:
   pipe, src, dest = 'meso', meso.ScanDone, ScanDone.Meso
else:
 PipelineException
```

Reply:
The for loop has more advantages.  The src is specified only once whereas in the if-elif-...-else structure, it has to be specified twice and that's error prone.  Same thing about spelling out pipe, src, dest multiple times.  I think the for loop is cleaner, especially if you add more pipelines.